### ISSUE TITLE
headphone and sizzle

### ISSUE THREAD

**Issue by JCx3812** [ngdanghau/DELL-latitude-E6540-macOS#2]
hello I get your efi partition and it works perfectly except when I connect my headphones I get a permanent sizzle. it's just me ? Do you have a solution ?

**Comment by ngdanghau** [comment]
Yeah. Actually, I got this when I install macOS, but later, I don't got this error again. So you use for a while. if not you can edit in config.plist by Clover Configurator in https://mackie100projects.altervista.org/download-clover-configurator/. 
Go to Boot and edit alcid=12 or alcid = 28 or 38

**Comment by ngdanghau** [comment]
remeber rebuild cache. it's helpful
Open Terminal and type all
sudo chmod -Rf 755 /S*/L*/E*
sudo chmod -Rf 755 /L*/E*
sudo chown -Rf 0:0 /S*/L*/E*
sudo chown -Rf 0:0 /L*/E*
sudo kextcache -i /
and restart
